### DISCUSSION TITLE
A브랜치의 머지 후 B브랜치 머지로 A 부분의 빌드가 안 되었는데(#71) 어떻게 비슷한 상황을 피하면 좋을까요?

### DISCUSSION THREAD

**Discussion by EuniceNam** [DeveloperAcademy-POSTECH/MacC-GetARock#73] [Q&A]
(#71: 컬러익스텐션 변경 후 이전 컬러익스텐션 코드가 작동 안함)

PR 승인 후 머지 전에 로컬에서 작업 브랜치로 develop을 머지해서 빌드하는 것은 품이 중복으로 과하게 드는 것 같은데,
이전 것에 영향을 미치는 PR은 따로 레이블을 붙여서 모두가 보게 한다거나 하면 될까요? 

들어보거나 생각나는 방법이 없어서, 궁금합니다

**Comment by unuhqueen** [comment]
저는 `develop` 브랜치가 업데이트 되면 무조건 내 브랜치로 merge하는게 가장 충돌이 안나는 좋은 방향이라고 생각합니다.
충돌이 나서 hotfix로 pr 날리고 수정하는게 더 품이 많이 드는 것 같아서요,,😅
